eql/parser.py

```diff
@@ -1068,10 +1084,16 @@ def _get_subqueries_and_close(self, node, allow_fork=False):
         close = None
 
         if until_node:
+            repeated_sequence = until_node["subquery_by"]["repeated_sequence"]
+            if repeated_sequence:
+                raise self._error(repeated_sequence, "Unsupported usage of repeated syntax", cls=EqlSyntaxError)
             subquery_nodes.append(until_node["subquery_by"])
 
         for pos, subquery in enumerate(subquery_nodes[1:], 1):
-            subqueries.append(self.subquery_by(subquery, num_values=num_values, allow_fork=allow_fork, position=pos))
+            subquery, join_values, runs = self.subquery_by(subquery, num_values=num_values, allow_fork=allow_fork,
+                                                           position=pos, allow_runs=allow_runs)
+            multiple_subqueries = [(subquery, join_values)] * int(runs)
```

Comment:
rather than casting this, should we just update `subquery_by` to return it as an `int`